pack-solution/action.yml

```diff
@@ -1,22 +1,59 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
-name: 'pack-solution'
-description: 'Power Platform Pack Solution'
+name: "pack-solution"
```

Comment:
why the churn to double quotes?

Reply:
Good question... looking the VS Code settings single quote is a non-default option. I updated mine to match the code. 
![image](https://user-images.githubusercontent.com/4356863/165102295-6aa19650-ba69-471d-a40c-9373370ab5e8.png)